src/test/java/org/acme/User/ServiceUserTest.java

```diff
@@ -62,16 +62,18 @@ public void testServiceUser() {
         modelUser.lastName = "Tyrrell";
 
         modelUser.cars.add(new ModelCar("ford", "Escort", "silver"));
+        modelUser.cars.get(0).user = modelUser;
```

Comment:
here i just manually set up the relationship.

in real code you would have either on user a addCar/removeCar method and/or on Car a setUser/clearUser that maintain the bidirectional relation ship..

But because of this complexity and how it also becomes a cyclic data structure one tend to only model one directional - which way is up to what makes the most sense for your code.

And then use queries to find "the other side".